Source/Core/DolphinWX/VideoConfigDiag.cpp

```diff
@@ -418,6 +418,12 @@ VideoConfigDiag::VideoConfigDiag(wxWindow* parent, const std::string &title, con
 	szr_enh->Add(CreateCheckBox(page_enh, _("Widescreen Hack"), wxGetTranslation(ws_hack_desc), vconfig.bWidescreenHack));
 	szr_enh->Add(CreateCheckBox(page_enh, _("Disable Fog"), wxGetTranslation(disable_fog_desc), vconfig.bDisableFog));
 
+	if (vconfig.backend_info.APIType == API_OPENGL)
```

Comment:
please create a flag if ssaa is supported, as it's almost trivial to also implement it for d3d.